Tasks/SonarQubePostTest/PRCA/ReportProcessor.ps1

```diff
@@ -2,6 +2,80 @@ $ProjectGuidAndFilePathMap = @{}
 $ComponentKeyAndPathMap = @{}
 $ComponentKeyAndRelativePathCache = @{}
 
+
+#region Public
+
+#
+# Fetches the new issues from the json report and also adds a "relativePath" for easier consumption
+#
+# Remark: the SonarQube report uses the component guid to refer to paths
+#
+function FetchAnnotatedNewIssues
+{    
+    $sonarReportFilePath = GetSonarReportFilePath
+    if (![System.IO.File]::Exists($sonarReportFilePath))
+    {
+        throw "Could not find the SonarQube issue report at $sonarReportFilePath. Unable to post issues to the PR."
+    }
+
+    Write-Verbose "Report file found at $sonarReportFilePath"    
+        
+    CreateProjectGuidAndPathMap    
```

Comment:
CreateProjectGuidToProjectPathMap might be better assuming that path that you refer to is the project path - not clear, and not documented at the top of the function.